crates/vm2/src/rollback.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::zero_sized_map_values)] // FIXME: why?
```

Comment:
Dumb question: Is there a particular reason `BTreeMap<_, ()>` is used instead of `BTreeSet`?

Reply:
I think RollbackableSet used to reuse RollbackableMap. Now that it is special-cased, it may as well use a set.